Automattic/jetpack
# VideoPress: fix removing and uploading video tracks on Simple sites

This PR fixes the issue when deleting or uploading a video track for the video block on Simple sites.

It uses the `isSimpleSite()` and `isAtomicSite()` helper functions to define what the `shouldUseJetpackVideoFetch()` returns and, according to it, choice the proper endpoint to hit the server side.

Fixes https://github.com/Automattic/jetpack/issues/27808
Fixes https://github.com/Automattic/jetpack/issues/27805

#### Changes proposed in this Pull Request:

* VideoPress: fix removing and uploading video tracks on Simple sites

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Create/edit a video block instance
* Try to upload a video track. Confirm it's doable after these changes
* Try to delete the track. Confirm it's doable after these changes
